bci_tester/data.py

```diff
@@ -31,19 +31,33 @@
 #: places
 OS_VERSION = os.getenv("OS_VERSION", "15.5")
 
-ALLOWED_OS_VERSIONS = ("15.3", "15.4", "15.5")
-_LANGUAGE_APPLICATION_STACK_OS_VERSIONS = ("15.4", "15.5")
+# Allowed os versions for base (non lang/non-app) containers
+ALLOWED_BASE_OS_VERSIONS = ("15.3", "15.4", "15.5")
 
-assert sorted(ALLOWED_OS_VERSIONS) == list(
-    ALLOWED_OS_VERSIONS
-), f"list ALLOWED_OS_VERSIONS must be sorted, but got {ALLOWED_OS_VERSIONS}"
+# Allowed os versions for Language and Application containers
+ALLOWED_NONBASE_OS_VERSIONS = ("15.4", "15.5")
 
-if OS_VERSION not in ALLOWED_OS_VERSIONS:
+# Test Language and Application containers by default for these versions
+_DEFAULT_NONBASE_OS_VERSIONS = ("15.5",)
+
+assert sorted(ALLOWED_BASE_OS_VERSIONS) == list(
+    ALLOWED_BASE_OS_VERSIONS
+), f"list ALLOWED_OS_VERSIONS must be sorted, but got {ALLOWED_BASE_OS_VERSIONS}"
+
+assert sorted(ALLOWED_NONBASE_OS_VERSIONS) == list(
+    ALLOWED_NONBASE_OS_VERSIONS
+), f"list ALLOWED_NONOS_VERSIONS must be sorted, but got {ALLOWED_NONBASE_OS_VERSIONS}"
```

Comment:
Why did you add this part? It's not wrong, but it's not user modifiable, so why?

Reply:
No particular reason, just to give hints to my future self on when the next times those settings/variables/constants are modified which rules they need to comply with.